onnxscript/_internal/ast_utils.py

```diff
@@ -22,3 +25,22 @@ def get_src_and_ast(f: types.FunctionType) -> tuple[str, ast.FunctionDef]:
     f_ast = top_level_ast.body[0]
     assert isinstance(f_ast, ast.FunctionDef)
     return src, f_ast
+
+
+def normalize_subscript_expr(expr: ast.Subscript):
+    # Normalizes the representation of a subscripted expression, handling python version
+    # differences as well as variations between A[x] (single-index) and A[x, y] (multiple indices)
+    # Returns a list of expressions, denoting the indices, after stripping the extraneous "Index"
+    # wrapper present in python versions before 3.9
+    index_expr = expr.slice
+    if py_version_ge_39:
+        if isinstance(index_expr, ast.Tuple):
+            return index_expr.elts  # multiple indices
+        else:
+            return [index_expr]  # single index
+    else:
+        if isinstance(index_expr, ast.ExtSlice):
+            indices = index_expr.dims  # multiple indices
+        else:
+            indices = [index_expr]  # single slice-index
```

Comment:
Looks like we are missing some coverage

Reply:
This is meant to handle differences between python 3.8 and 3.9. I assume that the code-coverage is running on one of the two versions. (Wonder if there is a way of combining code-coverage from multiple python versions!)

Reply:
hmm. I think codecov should combine coverage info from different runs